internals/parser.rst

```diff
@@ -92,7 +92,7 @@ Key ideas
   PEG parser doesn't have a concept of "where the :exc:`SyntaxError` is".
 
 
-Consequences or the ordered choice operator
+Consequences of the ordered choice operator
 -------------------------------------------
 
 .. _consequences-of-ordered-choice:
```

Comment:
```suggestion
.. _consequences-of-ordered-choice:

Consequences of the ordered choice operator
-------------------------------------------
```

While we're at it, we may as well fix the corresponding ref target label to actually point to the correct heading...